src/lib/cli/format.ts

```diff
@@ -192,7 +192,10 @@ export class RunningSprite {
 	}
 }
 
-export function getGlyphForStatus( status: StepStatus, runningSprite: RunningSprite ): string {
+export function getGlyphForStatus(
+	status: StepStatus | string,
```

Comment:
Because of this: https://github.com/Automattic/vip-cli/pull/1492/files#diff-e2db69e07a54f1d529171080f7d84e280b1dbb9b2e171ca55dc24e7e4fc571b4R207

We pass something that is not a `StepStatus` to this function.